pkg/common/constants/constants.go

```diff
@@ -64,6 +64,10 @@ const AnnotationTaskGroups = "yunikorn.apache.org/task-groups"
 const AnnotationSchedulingPolicyParam = "yunikorn.apache.org/schedulingPolicyParameters"
 const SchedulingPolicyTimeoutParam = "placeholderTimeoutInSeconds"
 const SchedulingPolicyParamDelimiter = " "
+const SchedulingPolicyStyleParam = "gangSchedulingStyle"
+const SchedulingPolicyStyleParamDefault = "Hard"
```

Comment:
this should be `Soft`
we need to keep this consistent with the core side: https://github.com/apache/incubator-yunikorn-core/blob/ef77f43d5c2829bedb12ed18578b8bf155793920/pkg/scheduler/objects/application.go#L107-L111, please remember to update the UT accordingly after this change.